src/js/engagement_view/src/components/graphDisplay/GraphDisplay.tsx

```diff
@@ -122,33 +121,46 @@ const GraphDisplay = ({ lensName, setCurNode }: GraphDisplayProps) => {
         [setHoverNode, updateHighlight, highlightLinks, highlightNodes]
     );
 
+    let clickedNodeKey = null;
+    if (clickedNode !== null) {
+        clickedNodeKey = clickedNode.id;
+    }
```

Comment:
I think this is the perfect use case for the Elvis Operator

```
let clickedNodeKey = clickedNode?.id;
```

same with below

Reply:
Skipping Elvis for now - because we have people who don't know TS well sometimes working in our TS codebase, I want the code to be as clear as possible even though Elvis is very very cool.